docs/user/metrics/memory_distribution.md

```diff
@@ -51,9 +55,44 @@ val snapshot = Memory.heapAllocated.testGetValue()
 assertEquals(11, snapshot.sum)
 
 // Usually you don't know the exact memory values, but how many should have been recorded.
-assertEquals(2L, snapshot.count())
+assertEquals(2L, snapshot.count)
 ```
 
+</div>
+
+<div data-lang="Swift" class="tab">
+
+```Swift
+func allocateMemory(nbytes: UInt64) {
+    // ...
+    Memory.heapAllocated.accumulate(nbytes / 1024)
+}
+```
+
+There are test APIs available too.  For convenience, properties `sum` and `count` are exposed to facilitate validating that data was recorded correctly.
+
+Continuing the `heapAllocated` example above, at this point the metric should have a `sum == 11` and a `count == 2`:
+
+```Swift
+@testable import Glean
+
+// Was anything recorded?
+XCTAssert(Memory.heapAllocated.testHasValue())
+
+// Get snapshot
+let snapshot = try! Memory.heapAllocated.testGetValue()
+
+// Does the sum have the expected value?
+XCTAssertEqual(11, snapshot.sum)
+
+// Usually you don't know the exact memory values, but how many should have been recorded.
+XCTAssertEqual(2, snapshot.count)
+```
+
+</div>
+
+{{#include ../../tab_footer.md}}
+
```

Comment:
We should start adding links to the Swift API docs for the class below in "Reference".  (And [this bug] https://bugzilla.mozilla.org/show_bug.cgi?id=1584990) can be used to do the backfill on that.

Reply:
yeah, let's do that in one swipe in the backfill